Modules/IO/LSM/src/itkLSMImageIO.cxx

```diff
@@ -273,8 +273,8 @@ LSMImageIO::Write(const void * buffer)
     return;
   }
 
-  uint32 w = width;
-  uint32 h = height;
+  uint2_t w = width;
```

Comment:
`uint32_t`. Compiler has eagle eyes 🖥️🦅👁️ 😄 

Similar mistake appears in a few more places, e.g.
`Modules\IO\TIFF\src\itkTIFFReaderInternal.cxx(149): error C2065: 'int2_t': undeclared identifier`